system.properties

```diff
@@ -0,0 +1 @@
+java.runtime.version=11
```

Comment:
do czego służą ci system.properties?

Reply:
to bylo potrzebne dla heroku zeby wiedzial jaka wersje ustawic javy, inaczej nie chcial lykac 11 wersje, ale i tak skonczylo ze odpalilem apke na VPS'ie, usune plik przy nastepnym pushu